orchestrator-salt/src/main/resources/salt/salt/postgresql/scripts/create_embeddeddb_certificate.sh

```diff
@@ -0,0 +1,23 @@
+{%- set postgres_fqdn = salt['grains.get']('fqdn') %}
+{%- set cm_keytab = salt['pillar.get']('keytab:CM') %}
+#!/usr/bin/env bash
+set -e
+
+function cleanup() {
+  kdestroy
+}
+
+trap cleanup EXIT
+
+CM_KEYTAB_FILE={{ cm_keytab.path }}
+CM_PRINCIPAL={{ cm_keytab.principal }}
+CERTS_DIR=/opt/certs/postgres
```

Comment:
Like above, one more point in favor of restricting DB SSL support only when the embedded DB is on the attached disk. :smile: